src/components/CurrentTask/CurrentTask.js

```diff
@@ -1,14 +1,29 @@
 import useStore from '../../hooks/useStore';
+import Button from '../Button/Button';
 
 import StyledCurrentTask from './StyledCurrentTask';
 
 export default function CurrentTask() {
 	const tasks = useStore(state => state.tasks);
+	const deleteTask = useStore(state => state.deleteTask);
 
-	return (
-		<StyledCurrentTask>
-			<p>{tasks[0].name}</p>
-			<p>{tasks[0].time} min</p>
-		</StyledCurrentTask>
-	);
+	function handleClick() {
+		deleteTask(tasks[0].id);
+	}
+
+	if (tasks.length) {
+		return (
+			<StyledCurrentTask>
+				<p>{tasks[0].name}</p>
+				<p>{tasks[0].time} min</p>
+				<Button onClick={handleClick}>done</Button>
+			</StyledCurrentTask>
+		);
+	} else {
+		return (
+			<StyledCurrentTask>
+				<p>Keine Aufgabe</p>
+			</StyledCurrentTask>
+		);
+	}
```

Comment:
You could also use a ternary operator here instead of an if/else statement, so it might be easier to read:

```suggestion
		return (
			<StyledCurrentTask>
			{tasks.length > 0 ? (
				<>
					<p>{tasks[0].name}</p>
					<p>{tasks[0].time} min</p>
					<Button onClick={handleClick}>done</Button>
				</>
			) : (
				<p>Keine Aufgabe</p>
			)}
		</StyledCurrentTask>
		);
```